extensions/amp-gfycat/0.1/amp-gfycat.js

```diff
@@ -90,13 +85,24 @@ class AmpGfycat extends AMP.BaseElement {
   createPlaceholderCallback() {
     const placeholder = this.win.document.createElement('amp-img');
     const videoid = dev().assertString(this.videoid_);
-
+    this.propagateAttributes(['alt', 'aria-label',
```

Comment:
Since you set `alt` and `aria-label` explicitly below, it is not necessary to propagate them here.  =) 

Since you propagate `aria-describedby` and `aria-labelledby`, can you also add unit tests that cover this behavior?